src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -39,4 +55,29 @@ public function getResponse() {
     return $this->response;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function missingDependencies() {
+    $capture_message = $this->t('Screen Capture package missing.');
```

Comment:
If you're suggesting keeping the messages in the utilities and just turning t() off, then that's probably okay.  We definitely should keep the messages themselves inside the plugins though as individual plugins should be responsible for generating their own error messages.